prac_05/emails.py

```diff
@@ -0,0 +1,30 @@
+"""
+Email
+Estimate: 20 minutes
+Actual:   40 minutes
+"""
+
+
+def main():
+    """Get email and extract name and store in dictionary."""
+    email_to_name = {}
+    email = input('Email: ')
+    while email != "":
+        name = extract_name(email)
+        choice = input(f'Is your name {name} (Y/n) ').upper()
```

Comment:
needs a question mark after {name} as it currently does not match the sample output

Reply:
Nice catch! 👍